src/main/java/de/embl/cba/mobie/MoBIE.java

```diff
@@ -66,407 +69,473 @@
     private String tableRoot;
     private HashMap<String, ImgLoader> sourceNameToImgLoader;
 
-    public MoBIE(String projectRoot) throws IOException {
-        this(projectRoot, MoBIESettings.settings());
+    public MoBIE( String projectRoot ) throws IOException
```

Comment:
It looks like you did some auto formatting on this file. 
In general it would be good not to do this mixed in with other PRs, because it makes reviewing very difficult because there are so many changes. Do you think you could revert the pure formatting changes without too much effort?

Reply:
Sorry for autoformatting. It will not happen again.